explanation/applications.md

```diff
@@ -9,6 +9,20 @@ To run on the Anbox Cloud platform, applications must fulfil the following requi
 
 If your application fulfils these requirements but you are still having issues running it on Anbox Cloud, file a [bug report](https://bugs.launchpad.net/indore-extern/+filebug).
 
+## Possible application statuses
+
+The following table lists the different statuses that an application can have depending on its state and what each status means:
+
+| Application status | Description |
+|--------------------|-------------|
+| `initializing` | The application is being created. Wait until the application reaches the `ready` state. |
+| `ready` | The application is successfully created and ready to be used. |
+| `deleted` | The application has been deleted and no longer available for use. |
+| `error` | The application has encountered an error. |
+| `unknown` | A possible error occurred and the real state of the application cannot be determined. |
+
+If you encounter the `error` or the `unknown` status, see if you can identify the base instance and troubleshoot using the instance logs (See [How to view the instance logs](https://discourse.ubuntu.com/t/how-to-view-the-instance-logs/24329)). If you are still unable to figure out the issue, [file a bug](https://bugs.launchpad.net/anbox-cloud) with the [relevant AMS logs](https://discourse.ubuntu.com/t/how-to-view-the-instance-logs/24329#view-stored-logs-2).
```

Comment:
The above link points people to view the stored instance logs rather than AMS logs, so 
with the [relevant AMS logs]
->
with the [relevant instance logs]

(In our documentation, we seem not to mention how to dump the AMS logs as end users usually won't need to do so. If there is a need, that probably means something bad happens. In that case, we could ask bug reporter to dump ams log further. however for this section, to me, that part can be omitted.)

Reply:
Simplified it to instance logs for applications and instances and just to file a bug for images and nodes.